instana/instrumentation/tornado/server.py

```diff
@@ -23,6 +23,18 @@
 
     setup_tornado_tracer()
 
+    def extract_custom_headers(span, headers):
+        if agent.options.extra_http_headers is None:
```

Comment:
Theoretically `headers` could also be None not just the `extra_http_headers`, plus strictly speaking the early return would make sense in a broader set of cases. Not just in case of None, but in case of anything Falsy, like empty collections etc. So the `not` operator could cover more.